packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -24,6 +26,38 @@ function cancelEvent(event) {
   event.stopPropagation()
 }
 
+function createPoint(event) {
+  const { clientX, clientY } = event
+  // SVG 2 uses DOMPoint
+  if (window.DOMPointReadOnly) {
+    return new DOMPointReadOnly(clientX, clientY)
+  }
+  // jsdom doesn't support SVG
+  return {
+    x: clientX,
+    y: clientY
+  }
+}
+
+function getEventOffset(event, canvas) {
+  const svgPoint = createPoint(event)
+  const svgEventOffset = svgPoint.matrixTransform
+    ? svgPoint.matrixTransform(canvas.getScreenCTM().inverse())
+    : svgPoint
+  return svgEventOffset
+}
+
+function convertEvent(event, canvas) {
+  const svgEventOffset = getEventOffset(event, canvas)
+  const svgCoordinateEvent = {
+    type: event.type,
+    x: svgEventOffset.x,
+    y: svgEventOffset.y
+  }
+
+  return svgCoordinateEvent
+}
```

Comment:
This change isn't necessary for the fix, but I thought it made the code easier to follow if it uses the same SVG transformation functions here as are used in the `draggable` decorator.

Reply:
In fact, the `InteractionLayer` component can import these functions, rather than defining them twice.